build-tools/scripts/JavaCallableWrappers.targets

```diff
@@ -28,9 +28,11 @@
       <_D>-d "$(IntermediateOutputPath)jcw\bin"</_D>
       <_AndroidJar>"$(AndroidToolchainDirectory)\sdk\platforms\android-$(AndroidPlatformId)\android.jar"</_AndroidJar>
       <_MonoAndroidJar>$(OutputPath)mono.android.jar</_MonoAndroidJar>
+      <_JarDelimiter Condition=" '$(HostOS)' != 'Windows' ">:</_JarDelimiter>
```

Comment:
We should move this into `Configuration.props` (topdir)`:

```xml
<PathSeparator>$([System.IO.Path]::PathSeparator)</PathSeparator>
```

This will set the value of `System.IO.Path.PathSeparator` into `$(PathSeparator)`, which can then be used instead of `:` or `;`.